main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/shirou/gopsutil/v3/disk"
+	"github.com/shirou/gopsutil/v3/mem"
+	"github.com/shirou/gopsutil/v3/process"
+)
+
+const (
+	_ = iota // Underscore means that it will be ignored.
+	// This creates a block of constants that will return
+	// 1 * 2^(10*iota). This formula is the description of
+	// bytes in the binary system used by computers.
+	KB = 1 << (10 * iota)
+	MB
+	GB
+	TB
+	PB
+	EB
+	ZB
+	YB
+)
+
+func main() {
+	fmt.Printf("\n------------ Memory info. ------------\n")
+	vm, _ := mem.VirtualMemory()
+	sm, _ := mem.SwapMemory()
+
+	fmt.Printf("Total: %.2f GB, Used: %.2f GB, Available: %.2f GB, UsedPercent: %.4f%%\n", float64(vm.Total)/GB, float64(vm.Used)/GB, float64(vm.Available)/GB, vm.UsedPercent)
+	fmt.Printf("Total: %.2f GB, Used: %.2f GB, Free: %.2f GB, UsedPercent: %.4f%%\n", float64(sm.Total)/GB, float64(sm.Used)/GB, float64(sm.Free)/GB, sm.UsedPercent)
+	fmt.Printf("\n------------ Memory info. ------------\n")
+
+	fmt.Printf("\n------------ Processes info. ------------\n")
+	fmt.Printf("\n---- PID | Name | CPU Percentage ----\n")
+	ps, _ := process.Processes()
+
+	// TODO: Filter all processes as they normally are many.
+	for _, p := range ps {
+		n, _ := p.Name()
```

Comment:
A number of these end up being blank for me.  You?\
<img width="323" alt="Screen Shot 2022-07-17 at 20 57 16" src="https://user-images.githubusercontent.com/611339/179439049-d3613597-fade-4730-a566-12313393b232.png">

Reply:
![image](https://user-images.githubusercontent.com/20165320/179439408-0633539d-3299-46a7-a7d1-c5943d9f257a.png)

Same, I believe is a expected behavior as there could be processes with very low usage or paused.